utils/__init__.py

```diff
@@ -264,10 +264,44 @@ def add_xmlns_sbml_attribute(sedml_filepath, sbml_filepath, output_filepath=None
     with open(output_filepath,"w") as fout:
         fout.write(sedstr)
 
+def add_xmlns_fbc_attribute(sedml_filepath, sbml_filepath, output_filepath=None):
+    '''
+    add an xmlns:sbml attribute to the sedml file that matches the sbml file
+    raise an error if the attribute is already present
+    output fixed file to output_filepath which defaults to sedml_filepath
+    '''
+    sedstr = ""
+    if output_filepath:
+        if os.path.exists(output_filepath):
+            with open(output_filepath, 'r') as file:
+                sedstr = file.read()    
+    if sedstr == "":
+        with open(sedml_filepath, 'r') as file:
+            sedstr = file.read()
+
+    m = re.search(r'<sedML[^>]*>', sedstr)
+    location = m.span()[1]-1
```

Comment:
You're calling m.span() before checking if m is None. I think the check needs to happen first otherwise it never catches the error because m.span() will throw its own error